src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,44 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exceptions.RegistrationException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final Integer REQUIRED_MIN_AGE = 18;
+    private static final Integer REQUIRED_MAX_AGE = 150;
```

Comment:
According to the checklist, avoid adding something not described in the task requirements, or tied to something that may change over time like max user age. The max age is an example of a value that may change as human lifespan changes, so it's better not to hardcode such values.

Reply:
ok, done